revoke/revoke_test.go

```diff
@@ -129,35 +166,68 @@ func mustParse(pemData string) *x509.Certificate {
 	return cert
 }
 
+func insertLocalCRL(revoke *Revoke) {
+	cert, err := x509.ParseCRL([]byte(localCRLList))
+	if err != nil {
+		panic(err.Error())
+	}
+	revoke.localCRL = localCRLPath
+	revoke.crlSet[localCRLPath] = cert
+}
+
 func TestRevoked(t *testing.T) {
-	if revoked, ok := VerifyCertificate(revokedCert); !ok {
+	rc := New(false)
+	if revoked, ok := rc.VerifyCertificate(revokedCert); !ok {
+		fmt.Fprintf(os.Stderr, "Warning: soft fail checking revocation")
+	} else if !revoked {
+		t.Fatalf("revoked certificate should have been marked as revoked")
+	}
+
+	insertLocalCRL(rc)
+	if revoked, ok := rc.VerifyCertificate(revokedCert); !ok {
 		fmt.Fprintf(os.Stderr, "Warning: soft fail checking revocation")
 	} else if !revoked {
 		t.Fatalf("revoked certificate should have been marked as revoked")
 	}
 }
 
 func TestExpired(t *testing.T) {
-	if revoked, ok := VerifyCertificate(expiredCert); !ok {
+	rc := New(false)
+	if revoked, ok := rc.VerifyCertificate(expiredCert); !ok {
+		fmt.Fprintf(os.Stderr, "Warning: soft fail checking revocation")
+	} else if !revoked {
+		t.Fatalf("expired certificate should have been marked as revoked")
+	}
+
+	insertLocalCRL(rc)
```

Comment:
this insertLocalCRL and test again isn't giving any more line coverage, restore it to the original state.